policy/aggregation_type_config.go

```diff
@@ -80,8 +61,13 @@ type AggregationTypesConfiguration struct {
 }
 
 // NewOptions creates a new Option.
-func (c AggregationTypesConfiguration) NewOptions(instrumentOpts instrument.Options) AggregationTypesOptions {
-	opts := NewAggregationTypesOptions()
+func (c AggregationTypesConfiguration) NewOptions(instrumentOpts instrument.Options) (AggregationTypesOptions, error) {
+	typeStringTypeTransformerFnType, err := parseTypeStringTransformerFnType(c.TypeStringTransformerType)
```

Comment:
If you make it an enum, you can just say `transformFn, err := c.TransformType.TransformFn`.